### PR TITLE
feat: The orderbook rejects orders with zero price

### PR DESCRIPTION
**Pull request by da-kami**
An order with zero price does not make any sense and should be rejected.

### CODE REVIEW

**Review by klochowicz — APPROVED** [review]

File: coordinator/src/orderbook/routes.rs [hunk 1]
```diff
@@ -82,6 +83,12 @@ pub async fn post_order(
     State(state): State<Arc<AppState>>,
     Json(new_order): Json<NewOrder>,
 ) -> Result<Json<Order>, AppError> {
+    if new_order.price == Decimal::ZERO {
```
Comment (bonomat) [inline coordinator/src/orderbook/routes.rs:86]: We could do `<=0` to also not allow negative prices.

**Review by bonomat — APPROVED** [review]

**Review by holzeis — APPROVED** [review]
Only downside to this fix is, that it is fixing a symptom rather than the issue. 🙂

**Comment by bonomat** [comment]
> Only downside to this fix is, that it is fixing a symptom rather than the issue. 🙂

What issue do you see?

**Pull request merged**